spec/support/acceptance_helper.rb

```diff
@@ -162,20 +162,20 @@ def initialize(callback, paths, *args)
     # Isolate collected changes between tests/listener instances
     @timed_changes = TimedChanges.new
 
-    if callback
-      @listener = Listen.send(*args) do |modified, added, removed|
-        # Add changes to trigger frozen Hash error, making sure lag is enough
-        _add_changes(:modified, modified, changes)
-        _add_changes(:added, added, changes)
-        _add_changes(:removed, removed, changes)
-
-        unless callback == :track_changes
-          callback.call(modified, added, removed)
-        end
-      end
-    else
-      @listener = Listen.send(*args)
-    end
+    @listener = if callback
+                  Listen.send(*args) do |modified, added, removed|
+                    # Add changes to trigger frozen Hash error, making sure lag is enough
+                    _add_changes(:modified, modified, changes)
+                    _add_changes(:added, added, changes)
+                    _add_changes(:removed, removed, changes)
+
+                    unless callback == :track_changes
+                      callback.call(modified, added, removed)
+                    end
+                  end
+                else
+                  Listen.send(*args)
+                end
```

Comment:
Adding the assignment in the next line could help in less indentation spaces.  
```suggestion
    @listener =
  if callback
    Listen.send(*args) do |modified, added, removed|
      # Add changes to trigger frozen Hash error, making sure lag is enough
      _add_changes(:modified, modified, changes)
      _add_changes(:added, added, changes)
      _add_changes(:removed, removed, changes)

      unless callback == :track_changes
        callback.call(modified, added, removed)
      end
    end
  else
    Listen.send(*args)
  end
```

Reply:
I also agree, this alignment change is a mess, and it should probably just be

```
@listener = if callback
  Listen.send(...)
else
  ...
end
```

Reply:
Rubocop wouldn't tolerate that formatting. I put 10 minutes into finding an option that would, but I didn't find it. Instead, I tried the `if` on the next line and it was ok with that. Of course then it complained that the method was too long! So I moved an `unless` from leading to trailing. 🤦 

https://github.com/guard/listen/pull/508/commits/745d0e35a433b5fe0bd8064ef6bdf0e5fc28e36b

Reply:
I think below should help. However as you have used next line, you won't need this then. Next lines looks good to me.
```yml
Layout/EndAlignment:
  EnforcedStyleAlignWith: variable
```

Reply:
Thanks @KapilSachdev. I made that `EnforcedStyleAlignWith: variable ` change and put the `@listener =` code back the way it was, then ran `rubocop -a -A`.  https://github.com/guard/listen/pull/508/commits/e1045d21d8cef8dd1752897f9d037a21a4e1f652